tools/pybind11Config.cmake.in

```diff
@@ -0,0 +1,91 @@
+# pybind11Config.cmake
```

Comment:
Added `share/cmake/pybind11/pybind11*cmake`, though that'll only catch the default install case. Hope that's right for you purposes.